src/components/interaction-area/interaction-area.utils.js

```diff
@@ -91,65 +91,60 @@ export const shouldShowAreaLayoutMessage = (interactionLayoutConfig) => {
   return (!interactionLayoutConfig || interactionLayoutConfig.enable) && localStorage.getItem(INTERACTION_AREA_LAYOUT_LOCAL_STORAGE_NAME) !== 'true';
 };
 
-export const createInteractionAreaLayoutMessage = (videojs, onClick) => {
+
+const onClickInteractionAreaLayoutClick = (checked, onClick) => {
+  localStorage.setItem(INTERACTION_AREA_LAYOUT_LOCAL_STORAGE_NAME, JSON.parse(checked));
+  onClick();
```

Comment:
if onClick might be null, add a check

Reply:
i will set noop as default :

`
const onClickInteractionAreaLayoutClick = (checked, onClick= noop) => {
`